src/Utilities/TypeTraits.hpp

```diff
@@ -993,10 +993,10 @@ struct is_streamable : std::false_type {};
 /// \cond HIDDEN_SYMBOLS
 template <typename S, typename T>
 struct is_streamable<
-    S, T, cpp17::void_t<decltype(std::declval<std::add_lvalue_reference_t<S>>()
-                                 << std::declval<T>()),
-                        Requires<not std::is_same<S, T>::value>>>
-    : std::true_type {};
+    S, T,
+    cpp17::void_t<decltype(std::declval<std::add_lvalue_reference_t<S>>()
```

Comment:
What is changing here and why? You can use `git clang-format -f` to format only the code you edited.

Reply:
Just clang formatting. I’ll use the -f flag from now on, to avoid edits just for formatting.